src/main/java/com/toptrumps/db/PersistentDataDAO.java

```diff
@@ -73,22 +73,34 @@ public static void updatePersistentData(){
       }
     }
 
-    public static ResultSet getStatistics(){
+    public static Statistics getStatistics(){
 
         Connection conn = ConnectionFactory.getConnection();
         PreparedStatement getStats = null;
         String getStatsSQL = "SELECT * FROM persistent_game_data;";
         ResultSet results = null;
+        Statistics statistics = null;
 
         try{
 
             getStats = conn.prepareStatement(getStatsSQL);
             results = getStats.executeQuery();
+
+            while(results.next()) {statistics = new Statistics(results);}
```

Comment:
I think `hasNext()` would be better here. `next()` returns the next object in results, and I don't think that's what you're trying to do here.

Reply:
https://www.tutorialspoint.com/java-resultset-next-method-with-example

Reply:
Why can't I pull your code? I'm pretty sure `hasNext` will work too.

Reply:
![image](https://user-images.githubusercontent.com/20328038/73867562-db248f80-483e-11ea-8ec1-fa6c7693588a.png)

Reply:
![image](https://user-images.githubusercontent.com/56762517/73868234-065bae80-4840-11ea-90eb-a7559dd56b8d.png)

Reply:
You win!